build.sbt

```diff
@@ -51,7 +51,7 @@ resolvers += "artifactory-releases" at artifactory + "libs-release"
 
 resolvers += "artifactory-snapshots" at artifactory + "libs-snapshot"
 
-val workbenchLibV = "d16cba9"
```

Comment:
Could you like to the relevant workbench lib PR/action that generates this, so we can double check this is the right version for the code fix?

Reply:
Yes it should be this one - https://github.com/broadinstitute/workbench-libs/pull/1681